stdlib/builtins.pyi

```diff
@@ -1188,7 +1188,7 @@ class property:
 class _NotImplementedType(Any):  # type: ignore[misc]
     # A little weird, but typing the __call__ as NotImplemented makes the error message
```

Comment:
Unrelated to your PR but we should probably revisit this and see if it actually makes things better for all type checkers.